app/js/event/event.sort.js

```diff
@@ -32,7 +32,7 @@
           getDistanceFromLatLonInKm(Number(events[i].latitude),
                                     Number(events[i].longitude),
                                     Number(coordinates.latitude),
-                                    Number(coordinates.longitude));
+                                    Number(coordinates.longitude)).toFixed(2);
```

Comment:
The event sort factory should not do any formatting.
It also looks like you already have `.toFixed()` in the filter?